src/main/scala/beam/utils/analysis/BlockGroupToTazMapper.scala

```diff
@@ -0,0 +1,73 @@
+package beam.utils.analysis
+
+import beam.sim.common.GeoUtils
+import beam.utils.ProfilingUtils
+import beam.utils.csv.CsvWriter
+import beam.utils.data.ctpp.models.ResidenceToWorkplaceFlowGeography
+import beam.utils.data.ctpp.readers.BaseTableReader.{CTPPDatabaseInfo, PathToData}
+import beam.utils.data.ctpp.readers.flow.HouseholdIncomeTableReader
+import beam.utils.data.synthpop.SimpleScenarioGenerator.getBlockGroupToTazs
+import beam.utils.data.synthpop.models.Models.{BlockGroupGeoId, TazGeoId}
+import beam.utils.data.synthpop.{GeoService, GeoServiceInputParam}
+
+import scala.util.Random
+
+object BlockGroupToTazMapper {
+  def quoteCsv(s: String): String = "\"" + s + "\""
+
+  def main(args: Array[String]): Unit = {
+    val pathToData = PathToData("D:/Work/beam/CTPP/")
+    // 34 - New Jersey
+    // 36 - New York
+    val databaseInfo = CTPPDatabaseInfo(pathToData, Set("34", "36"))
+    val readData =
+      new HouseholdIncomeTableReader(databaseInfo, ResidenceToWorkplaceFlowGeography.`TAZ To TAZ`)
+        .read()
+        .toVector
+
+    blockGroupToTazMapper("block_group_to_taz.csv")
+  }
+
+  private def blockGroupToTazMapper(pathToResult: String): Unit = {
+    val geoUtils: GeoUtils = new GeoUtils {
+      override def localCRS: String = "epsg:32118"
+    }
+
+    val geoSvc: GeoService = new GeoService(
+      GeoServiceInputParam(
+        "D:/Work/beam/NewYork/input/Shape/TAZ/",
```

Comment:
Should this be a param with defaults? especially using D:/

Reply:
Done